src/app/components/GenerateOperatorKeys/GenerateOperatorKeys.tsx

```diff
@@ -77,7 +89,19 @@ const GenerateOperatorKeys = () => {
       <Header title={translations.OPERATOR.REGISTER.TITLE} subtitle={translations.OPERATOR.REGISTER.DESCRIPTION} />
 
       <Grid container wrap="nowrap" spacing={0} className={classes.gridContainer}>
+        <br />
```

Comment:
Try to avoid `<br />` elements in such places, because they can be displayed with different heights in different browsers.
Instead, better to change the `classes.gridContainer` css